cloudsql-mysql-plugin/src/e2e-test/java/io/cdap/plugin/common/stepsdesign/TestSetupHooks.java

```diff
@@ -0,0 +1,155 @@
+package io.cdap.plugin.common.stepsdesign;
+
+import com.google.cloud.bigquery.BigQueryException;
+import io.cdap.e2e.utils.BigQueryClient;
+import io.cdap.e2e.utils.PluginPropertyUtils;
+import io.cdap.plugin.CloudMySqlClient;
+import io.cucumber.java.After;
+import io.cucumber.java.Before;
+import org.apache.commons.lang3.RandomStringUtils;
+import org.apache.commons.lang3.StringUtils;
+import org.junit.Assert;
+import stepsdesign.BeforeActions;
+
+import java.io.IOException;
+import java.nio.charset.StandardCharsets;
+import java.nio.file.Files;
+import java.nio.file.Paths;
+import java.sql.SQLException;
+import java.util.NoSuchElementException;
+import java.util.UUID;
+
+/**
+ * CLOUDSQL MYSQL test hooks.
+ */
+
+public class TestSetupHooks {
+  public static void setTableName() {
+    String randomString = RandomStringUtils.randomAlphabetic(10);
+    String sourceTableName = String.format("SourceTable_%s", randomString);
+    String targetTableName = String.format("TargetTable_%s", randomString);
+    PluginPropertyUtils.addPluginProp("sourceTable", sourceTableName);
+    PluginPropertyUtils.addPluginProp("targetTable", targetTableName);
+    PluginPropertyUtils.addPluginProp("selectQuery", String.format("select * from %s", sourceTableName));
+    System.out.println(sourceTableName);
```

Comment:
Remove this as well.

Reply:
done
commit link- https://github.com/cloudsufi/database-plugins/commit/b6c555b5f19974d2b06ebeb8708160feaa988973